web/src/containers/ApdExport.js

```diff
@@ -11,7 +11,7 @@ import { FileDownload } from '../components/Icons';
 const ExportAndSubmit = ({ printApd: print }) => (
   <Waypoint id="export-and-submit">
     <Section isNumbered id="export-and-submit" resource="exportAndSubmit">
-    <h3>Download</h3>
+    <h3 className="ds-h3">Download</h3>
```

Comment:
But you said this didn't address visual design...

![you fibbed a little](https://media.giphy.com/media/3ofT5EuCXAyVjR85RC/giphy.gif)

Reply:
just an eensy weensy bit